app/components/DeviceTimeModal.js

```diff
@@ -60,62 +60,52 @@ export class DeviceTimeModal extends Component {
   }
 
   getActions = () => {
+    const { showingDeviceTimePrompt: { cfg: { timezone }, times: { serverTime, deviceTime } } } = this.props;
+    const type = this.determineDeviceType();
     const buttons = [];
-    if ( !this.isAnimas() ) {
+    const footnote = type.value === 'bgm' ? '*' : '';
+    if ( !this.isDevice('Animas') &&
+         !this.isDevice('InsuletOmniPod') &&
+         !this.isDevice('Medtronic') &&     // these two lines should be removed
```

Comment:
I thought I saw a change in the 5/7 driver to set the time?

Reply:
Good catch! It's not working yet, so I should take it out.

Reply:
Good catch! I haven't gotten it to work yet, so should remove that code from the driver.